packages/web/src/components/gcds-select/gcds-select.tsx

```diff
@@ -180,13 +177,9 @@ export class GcdsSelect {
    */
   @Event() gcdsBlur!: EventEmitter<void>;
 
-  private onBlur = e => {
-    if (this.focusHandler) {
-      this.focusHandler(e);
-    } else {
-      if (this.validateOn == 'blur') {
-        this.validate();
-      }
+  private onBlur = () => {
+    if (this.validateOn == 'blur') {
```

Comment:
Could we take this opportunity to do a strict comparison?
```suggestion
    if (this.validateOn === 'blur') {
```